docs/source/error-codes/self-service/index.rst

```diff
@@ -191,7 +191,9 @@ but there is no guarantee given that additional information will be preserved ac
 Logging
 ---------------------------
 
-Generally, we use the following log-levels on the server:
+Errors reported over the gRPC are also written into the logfile. The communicated information is
+consistent between those two sources. A level at which errors are logged is determined by their
```

Comment:
Actually we transmit less over the wire (e.g. INTERNAL error logged information doesn't transpire over gRPC). Thinking better about it, I think that this entire section on logging can wait for 2.0 or when we are coherent on logging. 

e.g. We do not yet stand by the statements below - We are still logging in other places (no error codes) with `WARN` or `ERROR` what should be logged with `INFO`

Reply:
sounds good